cms/jinja2/templates/pages/release_calendar/release_calendar_page--cancelled.html

```diff
@@ -3,14 +3,12 @@
 {% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
 
 {% block release_meta %}
-    {% if page.release_date or page.release_text %}
-        <div class="ons-container common-header__releases common-header__releases--single-row">
-            <div>
-                <span class="ons-u-fs-r--b">{{ _("Release date") }}:</span>
-                <span class="ons-u-nowrap">{{ _("Cancelled") }}</span>
-            </div>
+    <div class="ons-container common-header__releases common-header__releases--single-row">
```

Comment:
note: #154 updates the heroes, so all templates changes will need updating once that is merged

Reply:
[595751](https://github.com/ONSdigital/dis-wagtail/pull/158/commits/65957513e647fc52a1bdc2ccd26d1d15fefb6892)